code/Test_definitions/LocationRetrieval.feature

```diff
@@ -0,0 +1,100 @@
+Feature: Get a device location (POST /retrieve)
+
+  @location_retrieval_01_by_phoneNumber_or_ipv4Address_or_networkAccessIdentifier
+  Scenario: Get the location of a device
+    Given a valid device <device> identified by a <deviceIdentierName> <deviceIdentierValue>
+    And a provided maxAge of 80 seconds
+    When a retrieve device location request is made 
+    Then the response status code should be 200

```

Comment:
A more assertive expression is better: "the response status code is XXX"

We have to define a set of expressions for these common steps in the Guidelines